CompanyEmployees.Infrastructure.Presentation/Controllers/AttendanceController.cs

```diff
@@ -0,0 +1,34 @@
+using CompanyEmployees.Application.Commands;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Shared.DataTransferObjects;
+
+namespace CompanyEmployees.Infrastructure.Presentation.Controllers;
+
+[Route("api/attendance")]
+[ApiController]
+public class AttendanceController : ControllerBase
+{
+    private readonly ISender _sender;
+
+    public AttendanceController(ISender sender)
+    {
+        _sender = sender;
+    }
+
+    /// <summary>
+    /// Records employee daily check-in
+    /// </summary>
+    /// <param name="attendanceForCreationDto">Attendance check-in data</param>
+    /// <returns>Created attendance record</returns>
+    [HttpPost("check-in")]
```

Comment:
Using `CreatedAtAction` instead of `Ok` would be more appropriate here to align with RESTful conventions when creating a resource. This improves clarity on what was created and provides a way to retrieve it via URL.

**Suggested change**
```suggestion
return CreatedAtAction("GetAttendanceById", new { id = attendance.Id }, attendance);
```